indico/modules/categories/forms.py

```diff
@@ -58,10 +58,14 @@ class CategorySettingsForm(IndicoForm):
                                                description=_("This message will show up at the top of every event page "
                                                              "in this category"))
     event_message = IndicoMarkdownField(_("Content"))
+    notify_managers = BooleanField(_("Notify managers"), widget=SwitchWidget(),
+                                   description=_("Whether to send email notifications to all managers of this category "
+                                                 "when an event is created inside it or in any of its subcategories."))
     event_creation_notification_emails = EmailListField(_("Notification E-mails"),
                                                         description=_("List of e-mails that will receive a notification"
```

Comment:
We usually use "emails", and with that change you can put the space at the end of this line instead of the beginning of the next one :)